docs/contributing.md

```diff
@@ -60,6 +60,10 @@ test suite you will need:
   * A `ruby` interpreter.
   * The [`FFI`](https://github.com/ffi/ffi) Ruby gem, installable via `gem install FFI`.
 
+We also support an environment variable UNIFFI_TESTS_DISABLE_EXTENSIONS; It is a set of file
+extensions in the format of your platform's PATH variable. Eg,
+`UNIFFI_TESTS_DISABLE_EXTENSIONS=swift:rb cargo test` will skip test filenames ending in `.swift`
+or `.rb`. (On Windows, use `swift;rb`)
```

Comment:
(In general I'm a [fan of semantic linebreaks](https://sembr.org/), that makes future changes much less noisy in diffs)

Reply:
More on context: huh, TIL Rust ships that as a function (`split_paths`), but I wonder if we should simply standardize on something here? Not sure how helpful the difference is per OS.
Then again it's a small feature so maybe not even worth spending time over.

Reply:
Yeah, I went around in circles here too, and just because I didn't like explaining it was different per-platform I just went with
"comma separated".